frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js

```diff
@@ -143,7 +146,9 @@ class AlgInputsLoadForecast extends Component {
                 res_l2_smooth: this.state.res_l2_smooth,
                 week_day: this.state.week_day,
                 publicl2_daily_use: parseFloat(this.state.publicl2_daily_use),
-                mixed_batteries: this.state.mixed_batteries,
+                small_batt: parseFloat(this.state.small_batt),
+                big_batt: parseFloat(this.state.big_batt),
+                all_batt: parseFloat(this.state.all_batt),
```

Comment:
You should really camelCase for your naming convention in JS. Snake casing is for python....

Reply:
You're right - fixed